gcd/src/main/java/org/spine3/server/storage/datastore/DsAggregateStorage.java

```diff
@@ -65,26 +67,37 @@
 public class DsAggregateStorage<I> extends AggregateStorage<I> {
 
     private static final String EVENTS_AFTER_LAST_SNAPSHOT_PREFIX = "EVENTS_AFTER_SNAPSHOT_";
+    private static final String AGGREGATE_TYPE = "aggregate_type";
 
     /**
-     * Prefix for the string IDs of the {@link AggregateStorageRecord records} which represent an aggregate snapshot,
+     * Prefix for the string IDs of the {@link AggregateEventRecord records} which represent an aggregate snapshot,
      * not an event.
      *
      * The aggregate snapshots are stored under an ID composed from {@code SNAPSHOT} and the aggregate ID.
      */
     private static final String SNAPSHOT = "SNAPSHOT";
 
-    private static final String KIND = AggregateStorageRecord.class.getName();
-    private static final String AGGREGATE_ENTITY_STATUS_KIND = EntityStatus.class.getName();
-    private static final TypeUrl TYPE_URL = TypeUrl.from(AggregateStorageRecord.getDescriptor());
+    private static final String KIND = TypeName.from(AggregateEventRecord.getDescriptor())
+                                               .value();
+    private static final String AGGREGATE_LIFECYCLE_KIND = TypeName.from(LifecycleFlags.getDescriptor())
+                                                                   .value();
+    private static final TypeUrl TYPE_URL = TypeUrl.from(AggregateEventRecord.getDescriptor());
```

Comment:
You can have only one instance of `TypeUrl` (instead of `KIND` above) and use conversion methods or `url.getTypeName()` at the place of usage.